rest-tests/src/test/resources/features/integration-db-to-db-metrics.feature

```diff
@@ -0,0 +1,22 @@
+@integrations-db-to-db-metrics
+Feature: Test functionality of monitoring metrics system, part 1.
+
+  Background: Clean application state
+    And remove all records from DB
```

Comment:
Shouldn't there be also general clean state?

Reply:
done @ pushed again